App.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom';
+import LoginPage from './LoginPage';
+
+class App extends React.Component {
+  render() {
+    return (
+      <Router>
```

Comment:
Consider wrapping your Routes inside a Switch component. This will render the first Route or Redirect that matches the location. If not, it might lead to unwanted behavior where multiple routes can match the same path.